lib/restful_model_collection.rb

```diff
@@ -15,16 +15,17 @@ def initialize(model_class, api, filters = {})
     def each
       return enum_for(:each) unless block_given?
 
-      offset = 0
-      chunk_size = 100
+      @filters[:offset] = 0
```

Comment:
Hmm, I'm not sure this is correct. When doing something like:
```ruby
messages.filter(:subject => 'Hey let's grab some coffee', :filter => 5).each do |msg|
    puts msg.subject
end````

Isn't your code going to get all messages, regardless of whether `filter` and `offset` were set?